optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/variable/descriptor/VariableDescriptor.java

```diff
@@ -28,7 +29,7 @@
 /**
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  */
-public abstract class VariableDescriptor<Solution_> {
+public abstract class VariableDescriptor<Solution_> implements Serializable {
```

Comment:
Interesting... What is the justification?

Reply:
Inherited classes are used in classes that implement Serializable. Should their fields that are not serializable be rather declared transient?

Reply:
Maybe those casses that use this and implement Serializable shouldn't implement Serializable?